website/templates/project/retracted_registration.mako

```diff
@@ -44,6 +44,13 @@
                 % else:
                     Not available
                 % endif
+                <span data-bind="if: hasIdentifiers()" class="scripted">
+                  <p>
+                    Identifiers:
+                  DOI <span data-bind="text: doi"></span>
+                      <span data-bind="if: hasArk()" class="scripted">| ARK <span data-bind="text: ark"></span></span>
```

Comment:
Based on the logic here and in `hasIdentifiers`, if a previously withdrawn registration had an ARK, but not a DOI, the ARK wouldn't be displayed here. Assuming that's a valid scenario in the identifiers world, let's fix it so the ARK shows.

Reply:
Talked with @caseyrollins in personal, there will never be a scenario that only ARK is available without DOI